Expensify/App
# Increase global create modal delay to 1500ms

### Details
Hopefully temporary workaround until https://github.com/Expensify/App/issues/5296 is resolved.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5303

### Tests / QA Steps

1. Log in with a user that's never signed into New Expensify before.
    - Dev-only tip: Emulate this by setting the following NVP:
    
        ```sql
        UPDATE nameValuePairs SET value = 'true' WHERE name = 'isFirstTimeNewExpensifyUser' AND accountID = INSERT_ACCOUNT_ID_HERE;
        ```
1. Ensure that the global create popover menu is correctly displayed.
1. Log back out and sign in with the same user.
1. The global create popover menu should not automatically open.

### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/47436092/134079633-f6d6fcac-d63c-43bc-bbce-4d6fb2cd1232.mov

#### Mobile Web
https://user-images.githubusercontent.com/47436092/134079965-e11da4bf-fdee-4b56-a006-1141280daf5f.mov

#### Desktop
https://user-images.githubusercontent.com/47436092/134080260-0cdff4f3-f54f-4bd5-85cb-c2af947fc810.mov

#### iOS
https://user-images.githubusercontent.com/47436092/134077604-814ac4c0-1944-4e25-801f-c10d9a177561.mov

#### Android
